src/Event.cs

```diff
@@ -62,6 +62,11 @@ public SubscriptionBuilder<EventContext> OnError(Action<EventContext> callback)
         }
 
         public SubscriptionHandle<EventContext> Subscribe(params string[] querySqls) => new(conn, Applied, Error, querySqls);
+
+        public void SubscribeToAllTables() {
```

Comment:
Hm wouldn't it be better to return the subcription handle from the API to give user some control if they need it?

Reply:
No, we explicitly want to give the user as little control as possible if they use this method. It's for simple apps where you don't care; if you want/need more control you should be writing out queries.

When we implement the new subscription API, we're going to keep this method in terms of "legacy subscriptions" (i.e. the current API). This will likely not play well with the `SubscriptionHandle` machinery.